tests/Koans20AboutTypeVariables.elm

```diff
@@ -0,0 +1,119 @@
+module Koans20AboutTypeVariables exposing (testSuite)
+
+import Expect
+import Utils.Blank exposing (..)
+import Utils.Test exposing (describe, test)
+
+
+
+{-
+   Function splitList works for lists of any item type.
+
+   The 'a' is a _type variable_. Type variables start with
+   lowercase characters.
+
+   There are, however, some reserved type variable names,
+   such as 'number', 'appendable', 'comparable' and 'compappend',
+   for _constrained type variables_.
+-}
+
+
+splitList : List a -> ( List a, List a )
```

Comment:
Can you kill the blank spaces between the documentation and the function that it documents? I think if you use the documentation comment syntax (`{-|`) elm-format will do this for you.

Reply:
It did - thanks for the hint.